src/main/java/com/github/vkorobkov/jfixturescmd/cmd/CmdParser.java

```diff
@@ -37,7 +39,12 @@ private JCommander buildJCommander() {
 
     private void generateSql() {
         String destination = cmdArgs.getDestination();
-        JFixturesResult result = JFixtures.byDialect(cmdArgs.getSource(), cmdArgs.getSqlType());
+        String fixtureFolder = cmdArgs.getSource();
+        SqlType sqlType = cmdArgs.getSqlType();
+
+        log.info("Fixture folder: " + fixtureFolder);
+        log.info("SQL type: " + sqlType);
+        JFixturesResult result = JFixtures.byDialect(fixtureFolder, sqlType);
         result.toFile(destination);
         log.info("The SQL file has bees successfully created: " + destination);
```

Comment:
let's add a blank line before the success message to make it more contrast and let's reformat it a bit like
```
Fixtures folder: /home/user/fixtures
SQL type: MySql

SUCCESS (destination file: /home/user/data.sql)
```